packages/message-system/src/config/config.v1.json

```diff
@@ -1,15 +1,15 @@
 {
     "version": 1,
-    "timestamp": "2023-03-07T00:00:00+00:00",
-    "sequence": 25,
+    "timestamp": "2023-04-05T00:00:00+00:00",
+    "sequence": 26,
     "actions": [
         {
             "conditions": [
                 {
                     "environment": {
-                        "desktop": ">=23.3.1",
+                        "desktop": ">=23.4.1",
```

Comment:
you have to update condition for disabling coinjoin on <23.4.1 too https://github.com/trezor/trezor-suite/pull/8015/files#diff-5ad5e06618040321eacdefdfb897ecec0bffe04e96f041df6a813563f22d4aeaR46
otherwise it will be available with default params for 23.3.x

Reply:
Ahh.. thx